recce/mcp_server.py

```diff
@@ -320,7 +343,14 @@ async def call_tool(name: str, arguments: Dict[str, Any]) -> List[TextContent]:
 
             try:
                 # Check if tool is blocked in non-server mode
-                blocked_tools_in_non_server = {"row_count_diff", "query", "query_diff", "profile_diff"}
+                blocked_tools_in_non_server = {
+                    "row_count_diff",
+                    "query",
+                    "query_diff",
+                    "profile_diff",
+                    "list_checks",
```

Comment:
can `list_checks` be allowed in non-server mode?

Reply:
Yes.. we should allow it.